package.json

```
@@ -74,7 +75,8 @@
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
     "test-api": "node test.js",
     "test-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 --reporter lcov node test.js",
-    "test": "npm run build && npm run format && npm run test-coverage"
+    "test-mutation": "npx stryker run",
```

`node_modules/.bin` is already in `PATH` in npm scripts, so there’s no need to use `npx` here.

```suggestion
    "test-mutation": "stryker run",
```